tests/unit/meta/Matches.ts

```diff
@@ -0,0 +1,137 @@
+import * as registerSuite from 'intern!object';
+import * as assert from 'intern/chai!assert';
+import { stub, SinonStub } from 'sinon';
+
+import global from '@dojo/shim/global';
+import sendEvent from '../../support/sendEvent';
+import { v } from '../../../src/d';
+import { ProjectorMixin } from '../../../src/main';
+import { WidgetBase } from '../../../src/WidgetBase';
+import { ThemeableMixin } from '../../../src/mixins/Themeable';
+
+import Matches from '../../../src/meta/Matches';
+
+let rAF: SinonStub;
+
+function resolveRAF() {
+	for (let i = 0; i < rAF.callCount; i++) {
+		rAF.getCall(i).args[0]();
+	}
+	rAF.reset();
+}
+
+registerSuite({
```

Comment:
Not to get 100% coverage on the code, as an invalid key is the same logic as a not-yet created key.